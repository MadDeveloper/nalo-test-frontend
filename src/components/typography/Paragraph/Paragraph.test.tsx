import renderer from "react-test-renderer"
import { Paragraph } from "."

describe("Components > Paragraph", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<Paragraph>Lorem ipsum</Paragraph>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
