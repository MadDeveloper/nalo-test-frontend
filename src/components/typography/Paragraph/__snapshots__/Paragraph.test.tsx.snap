// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Paragraph should match snapshot 1`] = `
<p>
  Lorem ipsum
</p>
`;
